docs/developer/plugin/external-plugin-functional-tests.asciidoc

```diff
@@ -64,7 +64,8 @@ export default async function ({ readConfigFile }) {
     }
 
     // more settings, like timeouts, mochaOpts, etc are
-    // defined in the config schema. See {blob}src/functional_test_runner/lib/config/schema.js[src/functional_test_runner/lib/config/schema.js]
+    // defined in the config schema. 
+    // See {kibana-blob}packages/kbn-test/src/functional_test_runner/lib/config/schema.ts
```

Comment:
@shainaraskas follow up on this path change

file was moved [here](https://github.com/elastic/kibana/commit/e62ec7c5ecf8b827f9489fca134fef5f247c911b?diff=split&w=0#diff-8bd25a026732eea583c9e4e16b28c2feb963131a8549a1316dad8960b12dd39d)

Reply:
Confirmed with the Kibana team that this new file is the correct replacement